data/data/gcp/bootstrap/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "bootstrap_instance" {
```

Comment:
the bootstrap module is deleted by the installer after bootstrapping completes, so any resource that uses these outputs will also be marked for deletion, therefore it's best if the outputs for bootstrap module is empty.